### PR TITLE
appveyor and nuget publish

### PR DESCRIPTION
**Pull request by adamchester**

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -4,7 +4,20 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <LangVersion>7.3</LangVersion>
     <TreatWarningsAsErrors Condition="'$(Configuration)' == 'Release'">true</TreatWarningsAsErrors>
-    <NoWarn>$(NoWarn);CS1591;CA1812;SA1600;SA1633;SA1200;SA1202;SA1201</NoWarn>
+    <NoWarn>$(NoWarn);CS1591;CA1812;SA1600;SA1633;SA1200;SA1202;SA1201;CA1016</NoWarn>
```
Comment (RajibLaha) [inline Directory.Build.props:7]: Reconsider `CA1016`

File: Directory.Build.props [hunk 2]
```diff
@@ -4,7 +4,20 @@
...
+    <Description>A client library for KMD Logic Audit</Description>
+    <VersionPrefix>0.0.1</VersionPrefix>
+    <Authors>KMD</Authors>
+    <PackageTags>kmd;logic</PackageTags>
+    <PackageIconUrl>https://console.kmdlogic.io/favicon-32x32.png</PackageIconUrl>
+    <PackageProjectUrl>https://github.com/kmdlogic/kmd-logic-audit-client</PackageProjectUrl>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <RepositoryUrl>https://github.com/kmdlogic/kmd-logic-audit-client</RepositoryUrl>
+    <RepositoryType>git</RepositoryType>
+    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
```
Comment (RajibLaha) [inline Directory.Build.props:20]: Reconsider `CA1016`

File: appveyor.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+artifacts:
+- path: artifacts/Kmd.Logic.Audit.Client*.nupkg
+deploy:
+- provider: NuGet
+  api_key:
+    secure: ZHHP6iRTV5jYf/5K8TGTLPcsjIizChJz2zZnx6JakCUSixeHAoLZC0ho5RT0pHnp
+  skip_symbols: true
+  on:
+    branch: /^(master)$/
+# - provider: GitHub
```
Comment (RajibLaha) [inline appveyor.yml]: Consider removing the commented codes or enable it.

File: build.ps1 [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+	Write-Host "build: Cleaning .\artifacts"
+	Remove-Item .\artifacts -Force -Recurse
+}
+
+# Get the branch from Appveyor or Travis or git
+$branch = $NULL
+if($env:APPVEYOR_REPO_BRANCH) {
+    $branch = $env:APPVEYOR_REPO_BRANCH
+}
+if($env:TRAVIS_BRANCH) {
```
Comment (RajibLaha) [inline build.ps1]: Consider check null on `$branch` before re-assign.

**Review by RajibLaha — CHANGES_REQUESTED** [review]

File: Directory.Build.props (hunk 1 above)
Comment (adamchester) [reply Directory.Build.props:7]: For now I think we should try to leave `CA1016` disabled and `GenerateAssemblyVersionAttribute` set to `false` due to similar issues described here: https://github.com/dotnet/sdk/issues/1450

File: Directory.Build.props (hunk 2 above)
Comment (adamchester) [reply Directory.Build.props:20]: For now I think we should try to leave `CA1016` disabled and `GenerateAssemblyVersionAttribute` set to `false` due to similar issues described here: https://github.com/dotnet/sdk/issues/1450

**Review by RajibLaha — APPROVED** [review]

**Pull request merged**